pkg/controller/installation/validation.go

```diff
@@ -143,6 +143,13 @@ func validateCustomResource(instance *operatorv1.Installation) error {
 			instance.Spec.FlexVolumePath)
 	}
 
+	if instance.Spec.Variant == operatorv1.TigeraSecureEnterprise {
```

Comment:
I think you mean to check != or need to change to operatorv1.Calico

Reply:
doh yeah, fixed.